src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/CachedInterfaceDispatch.cs

```diff
@@ -20,6 +20,7 @@ private static unsafe IntPtr RhpCidResolve(IntPtr callerTransitionBlockParam, In
             return dispatchResolveTarget;
         }
 
+        [RuntimeExport("RhpCidResolveWasm")]
```

Comment:
I did originally write it that way, but it looked ugly:
```
#if TARGET_WASM
        [RuntimeExport("RhpCidResolveWasm")]
#endif
        private static IntPtr RhpCidResolve_Worker(...)
```
Since the export name makes it pretty clear this is Wasm-only functionality, I decided to just delete the `#if`.